temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
+print("hello")
```

Comment:
Avoid using hardcoded strings in the code. If this message is for debugging, consider using logging instead.
```
import logging

# Set up logging configuration
logging.basicConfig(level=logging.INFO)

# Use logging instead of print for output
logging.info('hello')
```